bin/auth0_create.js

```diff
@@ -0,0 +1,142 @@
+                type: 'boolean',
+            },
+            'bundle-minify': {
+                description: 'Generate a minified production build',
+                type: 'boolean',
+                dest: 'minify'
+            },
+            'bundle-strict': {
+                description: 'Enforce strict semver matching for bundling with `webtask-bundle`',
+                dest: 'loose',
+                action: 'storeFalse',
+                defaultValue: true,
+                type: 'boolean',
+            },
+            'capture': {
+                description: 'Download and use the current code indicated by `url`. When you are developing a webtask whose code is remotely hosted, this option will automatically download the remote code before creating the webtask. This means that the webtask will continue to run even if the remote url becomes unavailable.',
+                type: 'boolean',
+            },
+        },
+    },
+    params: {
+        'file_or_url': {
+            description: 'Path or URL of the extension\'s code. When not specified, code will be read from STDIN.',
+            type: 'string',
+        },
+    },
+    handler: handleCreate,
+});
+
+// Command handler
+
+function handleCreate(args) {
+    args = ValidateCreateArgs(args);
+    var profile = args.profile;
+
+    return profile.inspectWebtask({ 
+        name: args.name, 
+        meta: true 
+    })
+    .then(function (claims) {
+        if (!claims.meta || claims.meta['auth0-extension'] !== 'runtime')
+            throw Cli.error.invalid('Webtask ' + args.name + ' exists but is not an Auth0 hook. Please specify a different name using --name parameter.');
+        if (claims.meta['auth0-extension-name'] !== args.extensionName)
+            throw Cli.error.invalid('Auth0 hook ' + args.name + ' exists but is of type ' + claims.meta['auth0-extension-name'] + '. Please specify a different name using --name parameter to avoid a conflict.');
+        return _create(claims);
+    }, function (e) {
+        if (e && e.statusCode !== 404) {
+            throw e;
+        }
+        return _create();
+    });
+
+    function _create(claims) {
+        args.params = [];
+        args.meta['auth0-extension'] = 'runtime';
+        args.meta['auth0-extension-name'] = args.extensionName;
+
+        // If updating exising hook, preserve the enabled/disabled state
+        if (!claims || claims.meta['auth0-extension-disabled'])
+            args.meta['auth0-extension-disabled'] = '1';
+
+        // If wt-compiler specified explicitly, use it. Otherwise use the default per hook type.
+        if (!args.meta['wt-compiler'])
+            args.meta['wt-compiler'] = require('./auth0_extensions')[args.extensionName].compiler;
```

Comment:
I will be ripping it out and taking dependency on https://github.com/auth0/auth0-hooks-templates/pull/1 instead